noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -168,56 +168,6 @@ pub global REGISTERER_UNCONSTRAINED_FUNCTION_BROADCASTED_MAGIC_VALUE: Field =
 pub global DEPLOYER_CONTRACT_INSTANCE_DEPLOYED_MAGIC_VALUE: Field =
     0x85864497636cf755ae7bde03f267ce01a520981c21c3682aaf82a631;
 
-// GAS DEFAULTS
-pub global DEFAULT_GAS_LIMIT: u32 = 1_000_000_000;
-pub global DEFAULT_TEARDOWN_GAS_LIMIT: u32 = 12_000_000;
-pub global MAX_L2_GAS_PER_ENQUEUED_CALL: u32 = 12_000_000;
-pub global DA_BYTES_PER_FIELD: u32 = 32;
-pub global DA_GAS_PER_BYTE: u32 = 16;
-// pays for preamble information in TX Effects
-pub global FIXED_DA_GAS: u32 = 512;
-// pays for fixed tx costs like validation, and updating state roots
-pub global FIXED_L2_GAS: u32 = 512;
-// base cost for a single public call
-pub global FIXED_AVM_STARTUP_L2_GAS: u32 = 1024;
```

Comment:
i wonder why the diff thinks you changed these lines?

Reply:
I moved some gas related constants further down the file because of the Poseidon base L2 gas value which needed to be before. Maybe that is the cause of the strange diff.